src/nft/ZoraCreator1155Impl.sol

```diff
@@ -481,7 +482,7 @@ contract ZoraCreator1155Impl is
         }
 
         totalRoyaltyMints = (mintAmount + (tokens[tokenId].totalMinted % royaltyMintSchedule)) / (royaltyMintSchedule - 1);
-
+        totalRoyaltyMints = MathUpgradeable.min(totalRoyaltyMints, tokens[tokenId].maxSupply - (mintAmount + tokens[tokenId].totalMinted));
```

Comment:
as @iainnash suggested, I'd cache these storage reads in a local variable, to prevent the possibility it's double reading from storage.

Reply:
@jgeary do you have a chance to update this?

Reply:
@jgeary ping here